### ISSUE TITLE
doesn't do anything

### ISSUE THREAD

**Issue by RivaldisR** [xabier1989/WiredPlayers-RP#24]
everything starts but nothing happens when i am logging to the server

https://imgur.com/a/qI8hQPg

**Comment by RivaldisR** [comment]
Sorry it didn't downloaded any resources it just said downloading resources and download completed straight away

**Comment by xabier1989** [comment]
And the required ports are opened right? Did you see the "downloading packages" message on the bottom right of your screen?

**Comment by RivaldisR** [comment]
No it didn't showed up it just said on the left bottom screen that connected downloading resources and straight away downloaded resources also is this might me an error ?

`==================[11/5/2018 9:27:04 AM]==================
System.IO.FileNotFoundException: Could not load file or assembly 'C:\RAGEMP\server-files\bridge\resources\WiredPlayers\bin\Debug\netcoreapp2.0\WiredPlayers.resources.dll'. The system cannot find the file specified.
File name: 'C:\RAGEMP\server-files\bridge\resources\WiredPlayers\bin\Debug\netcoreapp2.0\WiredPlayers.resources.dll'
   at void System.Runtime.Loader.AssemblyLoadContext.LoadFromPath(IntPtr ptrNativeAssemblyLoadContext, string ilPath, string niPath, ObjectHandleOnStack retAssembly)
   at Assembly System.Runtime.Loader.AssemblyLoadContext.LoadFromAssemblyPath(string assemblyPath)


=========================================================
`

**Comment by xabier1989** [comment]
Yes, that's the main problem, you didn't compile the gamemode at Visual Studio.

**Comment by RivaldisR** [comment]
I did 
![image](https://user-images.githubusercontent.com/43607804/47990270-0752dd00-e0df-11e8-9fe4-a212c472409a.png)

**Comment by RivaldisR** [comment]
and that's what I get 
![image](https://user-images.githubusercontent.com/43607804/47990474-82b48e80-e0df-11e8-90c3-20bac0b8bd8e.png)

**Comment by RivaldisR** [comment]
but it doesn't download anything

**Comment by xabier1989** [comment]
Then check that the .dll is really on that path and launch the server back. If the path and name file are the same as it seems, it should not throw any error.

**Comment by RivaldisR** [comment]
![image](https://user-images.githubusercontent.com/43607804/47990603-d6bf7300-e0df-11e8-8492-eee6055e9967.png)

**Comment by RivaldisR** [comment]
nothing

**Comment by xabier1989** [comment]
Still the missing .dll error? If so, do you have execute permission on that folder? Maybe try running server.exe with admin rights.

**Comment by RivaldisR** [comment]
cleared logs and started server
[server_exceptions.txt](https://github.com/xabier1989/WiredPlayers-RP/files/2547804/server_exceptions.txt)
[server_logs.txt](https://github.com/xabier1989/WiredPlayers-RP/files/2547806/server_logs.txt)

**Comment by xabier1989** [comment]
Will check that with a clean installation and see what could be wrong.

**Comment by xabier1989** [comment]
I can't see anything, did you edit the meta.xml? Because the .dll file should be Wiredplayers.dll, not Wiredplayers.resource.dll

Also if this doesn't work, try downloading the version uploaded to rage.mp forum.

**Comment by RivaldisR** [comment]
![image](https://user-images.githubusercontent.com/43607804/48047223-4d677980-e18f-11e8-8e0c-c219e6c18f07.png)

**Comment by RivaldisR** [comment]
HAHA the one from rage.mp works correctly

**Comment by RivaldisR** [comment]
but the error with driving school not fixed

**Comment by xabier1989** [comment]
Did you fill the required tables into the database? It's more explained on the FAQ.

**Comment by RivaldisR** [comment]
Yeah it's working fine. When you gonna fix this version error ?

**Comment by xabier1989** [comment]
It's about the language resources not being compiled into .dll, so I'll just revert it when I have time. Meanwhile you can just use the previous version which has the same content.

**Comment by Mafios** [comment]
i have the same error..how can i solve it?

**Comment by Ownness** [comment]
> It's about the language resources not being compiled into .dll, so I'll just revert it when I have time. Meanwhile you can just use the previous version which has the same content.

are you fixed it?
I have the same error..how can I solve it?
